src/diffusers/pipelines/stable_diffusion/__init__.py

```diff
@@ -39,6 +39,7 @@ class StableDiffusionPipelineOutput(BaseOutput):
 if is_transformers_available() and is_torch_available():
     from .pipeline_cycle_diffusion import CycleDiffusionPipeline
     from .pipeline_stable_diffusion import StableDiffusionPipeline
+    from .pipeline_stable_diffusion_depth2img import StableDiffusionDepth2ImgPipeline
```

Comment:
Might need the version hack here: `is_transformers_available() and is_torch_available() and is_transformers_version(">=", "4.26.0.dev0")`